addons/website_sale_stock_wishlist/views/website_sale_stock_wishlist_templates.xml

```diff
@@ -46,7 +46,8 @@
         <xpath expr="//button[@id='add_to_cart_button']" position="replace">
             <t t-set="is_sold_out"
                t-value="not wish.product_id.allow_out_of_stock_order and wish.product_id._is_sold_out()"/>
-            <button id="add_to_cart_button"
+            <t t-set="combination_info" t-value="wish.product_id._get_combination_info_variant()"/>
```

Comment:
as far as I can see, we already have combination_info at this line:

https://github.com/odoo/odoo/blob/cedd0603a24aa0c0e6716c2e0f49e89c870eae04/addons/website_sale_wishlist/views/website_sale_wishlist_template.xml#L178

the view is rendered with all override applied, so you can use the value inside the parent view